data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/berserk/berserk_different_values.cfg

```diff
@@ -0,0 +1,45 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [berserk]value=
+##
+# Actions:
+# Give the side 1 leader a berserk ability with a value of 5
+# Give the side 2 leader a berserk ability with a value of 7
+# Have the side 1 leader attack the side 2 leader with both its weapons
+##
+# Expected end state:
+# 7 rounds of combat happens with the first weapon since the opponent has a berserk with a higher value
+# 5 rounds of combat happens with the second weapon since the opponent has no ranged attack
+#####
+{COMMON_KEEP_A_B_UNIT_TEST "berserk_different_values" (
+    [event]
+        name = start
+
+        {SET_HP VALUE=100}
+        [modify_unit]
+            [filter]
+                side=1
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY berserk 5 () SELF=yes}
+                [/abilities]
+            [/effect]
+        [/modify_unit]
+        [modify_unit]
+            [filter]
+                side=2
+            [/filter]
+            [effect]
+                apply_to = new_ability
+                [abilities]
+                    {TEST_ABILITY berserk 7 () SELF=yes}
+                [/abilities]
+            [/effect]
+        [/modify_unit]
+    [/event]
+
+    {CHECK_STRIKES "14,20" ({SUCCEED})}
```

Comment:
Having this test succeed on death means it'll intermittently pass even if the code under test is broken.